cmd/security-agent/main_windows.go

```diff
@@ -61,19 +61,6 @@ func main() {
 	flavor.SetFlavor(flavor.Dogstatsd)
 	config.Datadog.AddConfigPath(DefaultConfPath)
 
-	// go_expvar server
-	go func() {
-		var port = coreconfig.Datadog.GetString("security_agent.expvar_port")
-		coreconfig.Datadog.Set("expvar_port", port)
-		if coreconfig.Datadog.GetBool("telemetry.enabled") {
-			http.Handle("/telemetry", telemetry.Handler())
-		}
-		err := http.ListenAndServe("127.0.0.1:"+port, http.DefaultServeMux)
-		if err != nil && err != http.ErrServerClosed {
-			log.Errorf("Error creating expvar server on port %v: %v", port, err)
-		}
-	}()
-
```

Comment:
Remove this code since the code below runs go_expvar server already.
https://github.com/DataDog/datadog-agent/blob/ec89f1b6645948fe2ae895180b85606b2da4d3a4/cmd/security-agent/app/app.go#L231-L242